build/xtask/src/config.rs

```diff
@@ -27,6 +27,8 @@ struct RawConfig {
     epoch: u32,
     #[serde(default)]
     version: u32,
+    #[serde(default)]
+    fwid: bool,
```

Comment:
Would it be easier to make this `no_fwid` so only boards that we probably aren't going to care about can opt out? I was trying to go through the mental list to make sure we hit all the boards we care about.

Reply:
I went back and forth on this a few times. The current approach, as you point out, is error prone & requires we update more toml files. Doing this with negative logic would require fewer toml changes but might end up on the wrong side of the law of least surprise when folks add new boards and they get build errors until they figure out that they need to add `fwid = false`. There are good reasons for going both ways. I'm happy to defer to your judgement.

Reply:
Eh, let's keep it as is for now.